omnibus/files/private-chef-cookbooks/private-chef/templates/default/pedant_config.rb.erb

```diff
@@ -61,7 +61,11 @@ default_orgname <%= @default_orgname ? "\"#{@default_orgname}\"" : "nil" %>
 # testing location, you should not specify a value for this parameter.
 # The tests will still run, albeit slower, as they will now need to
 # poll for a period to ensure they are querying committed results.
+<% if node['private_chef']['search_provider'] == "solr" %>
 search_server "<%= @solr_url %>"
+<% else %>
+# search_server "<%= @solr_url %>"
```

Comment:
Nope!  This is a bit convoluted but is explained a bit by the comment above.  When search_server isn't present, what changes is how we poll for search changes.  When it is present we send an explicit commit and then assume that the next search will have up to date result.  When it isn't present we repeat the search over and over until the test passes or we time out.  CloudSearch doesn't use the explicit commit so we have to do the latter, which is much much slower.